src/domain/fiat/display-currency.ts

```diff
@@ -0,0 +1,5 @@
+export const toDisplayCurrency =
+  (price: UsdPerSat) =>
```

Comment:
Price here should probably be `DisplayCurrencyConversionRate` or something like that.

Reply:
Also why currying? Cannot find a place in the code where it is needed.

Reply:
the idea is that the price will be a "config" of this (to be expanded) function. this is why it's curried.